src/modules/map/components/LocationPopup.vue

```diff
@@ -157,7 +157,7 @@ export default {
         },
         coordinateWGS84Plain() {
             const wgsMetric = this.coordinateWGS84Metric
-            return `${round(wgsMetric[1], 5)}, ${round(wgsMetric[0], 5)}`
+            return `${round(wgsMetric[1], 5, true)}, ${round(wgsMetric[0], 5, true)}`
```

Comment:
I would apply this forceDigit to all outputs of this LocationPopup component for consistency (so all other coordinate systems with floating numbers)